src/Nethermind/Nethermind.Consensus/IGasLimitCalculator.cs

```diff
@@ -8,5 +8,6 @@ namespace Nethermind.Consensus
     public interface IGasLimitCalculator
     {
         long GetGasLimit(BlockHeader parentHeader);
+        long GetGasLimit(BlockHeader parentHeader, long? targetGasLimit);
```

Comment:
I think there should be only one of these methods. Make targetGasLimit `null` by default.